tests/install/openqa_worker.pm

```diff
@@ -5,8 +5,9 @@ use utils;
 
 sub run {
     diag('worker setup');
-    assert_script_run('zypper --no-cd --non-interactive --gpg-auto-import-keys in os-autoinst',   600);
-    assert_script_run('zypper --no-cd --non-interactive --gpg-auto-import-keys in openQA-worker', 600);
+    assert_script_run('zypper -n --gpg-auto-import-keys ref -f',                                  60);
```

Comment:
We don't have an automatic tidy check in this repo I assume but I doubt we need to keep the line like this. Looks weird if the timeout argument is separated by this many blanks. Also I suggest to just leave out the timeout and rely on the default which is 90s

Reply:
Oops, that was a leftover. Fixed.